plugins/dynatrace/README.md

```diff
@@ -17,6 +21,39 @@ The Dynatrace plugin will require the following information, to be used in the c
   - `entities.read`
   - `problems.read`
 
+#### Install
+
+1. Install the plugin on your frontend:
+
+```
+yarn add --cwd packages/app @backstage/plugin-dynatrace
+```
+
+2. We created in our catalog the interface for using the integration with Dynatrace.
+
+```diff
+# packages/app/src/components/catalog/EntityPage.tsx
+
+[...]
++ import { DynatraceTab, isDynatraceAvailable } from '@backstage/plugin-dynatrace'
+
+[...]
+
+const serviceEntityPage = (
+  <EntityLayout>
+    [...]
+    <EntityLayout.Route
+      path="/dynatrace"
+      title="Dynatrace"
+      if={isDynatraceAvailable}
+    >
+      <DynatraceTab />
+    </EntityLayout.Route>
```

Comment:
Missed this the last time around, lines 45 to 51 should all begin with the `+` symbol to show they were added

Reply:
Perfect!